server/src/app/controllers/taskController.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response } from 'express'
```

Comment:
This should be part of a presentation layer. Instead of `controller` you can call it `presentation` because a controller is something very specific, but a presentation is a layer with more stuff.

Reply:
It also shouldn't be called `app`. remove the word `app` from the directory.

Reply:
Thanks, I have remove it